.github/actions/package/action.yml

```diff
@@ -58,6 +61,11 @@ runs:
           export DIST=".${{ inputs.distrib }}"
         else
           export DIST=""
+          if [ "${{ inputs.stability }}" = "unstable" ] || [ "${{ inputs.stability }}" = "canary" ]; then
+            export RELEASE="$RELEASE-${{ inputs.distrib }}"
+          elif [ "${{ inputs.stability }}" = "testing" ]; then
+            export RELEASE="${{ inputs.distrib }}"
```

Comment:
@tuntoja should we add a release number for testing repo ? ex: `1-${{ inputs.distrib }}`

Reply:
I think we can, since there will always only be a single testing package, it should be ok